### ISSUE TITLE
Update .jslintrc to allow ECMA ES6

### ISSUE THREAD

**Issue by Tardo** [OCA/pylint-odoo#259]
js-linter options needs be updated to don't show as error the es6 syntax.

@moylop260

**Comment by moylop260** [comment]
Hi @Tardo 
Could you share us the steps to reproduce it, please?

**Comment by Tardo** [comment]
Hi @moylop260,
I'm using https://github.com/OCA/pylint-odoo/blob/master/pylint_odoo/examples/.jslintrc on local environment with Atom... if I not add ```"ecmaVersion": 6``` on ```"parserOptions"``` the linter shows me ES6 syntax as error.

P.S: Perhaps needs touch other rules... i only made the minimal changes to work properly.

**Comment by moylop260** [comment]
Could you create a pull request to change [pylint_odoo/examples/.jslintrc](https://github.com/OCA/pylint-odoo/blob/master/pylint_odoo/examples/.jslintrc), please?

**Comment by Tardo** [comment]
Ok, no problem.

**Comment by rafaelbn** [comment]
@moylop260 here you are my friend: https://github.com/OCA/pylint-odoo/pull/260
